content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -0,0 +1,182 @@
+
+    Ask for sponsorship in the [#sig-docs channel on Slack](https://kubernetes.slack.com) or on the
+    [SIG Docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
+
+    {{< note >}}
+    Don't send a direct email or Slack direct message to an individual
+    SIG Docs member. You must request sponsorship before submitting your application.
+    {{< /note >}}
+
+2.  Open a GitHub issue in the [`kubernetes/org`](https://github.com/kubernetes/org/) repository. Use the **Organization Membership Request** issue template.
+
+3.  Let your sponsors know about the GitHub issue. You can either:
+  - Mention their GitHub username them in an issue (`@<GitHub-username>`)
+  - Send them the issue link using Slack or email.
+
+  Sponsors will approve your request with a `+1` vote. Once your sponsors approve the request, a Kubernetes GitHub admin adds you as a member. Congratulations!
+
+  If your membership request is not accepted you will receive feedback. After addressing the feedback, apply again.
+
+4. Accept the invitation to the Kubernetes GitHub organization in your email account.
+
+  {{< note >}}
+  GitHub sends the invitation to the default email address in your account.
+  {{< /note >}}
+
+## Reviewers
+
+Reviewers are responsible for reviewing open pull requests. Unlike member feedback, you must address reviewer feedback. Reviewers are members of the [@kubernetes/sig-docs-{language}-reviews](https://github.com/orgs/kubernetes/teams?query=sig-docs) GitHub team.
+
+Reviewers can:
+
+- Do everything listed under [Anyone](#anyone) and [Members](#members)
+- Review pull requests and provide binding feedback
+- Edit user-facing strings in code
+- Improve code comments
+
+You can be a SIG Docs reviewer, or a reviewer for docs in a specific subject area.
+
+### Assigning reviewers to pull requests
+
+Automation assigns reviewers to all pull requests. You can request a
+review from a specific person by commenting: `/assign
+[@_github_handle]`.
+
+If the assigned reviewer has not commented on the PR, another reviewer can step in. You can also assign technical reviewers as needed.
+
+### Using `/lgtm`
+
+LGTM stands for "Looks good to me" and indicates that a pull request is technically accurate and ready to merge. All PRs need a `/lgtm` comment from a reviewer and a `/approve` comment from an approver to merge.
+
+A `/lgtm` comment from reviewer is binding and triggers automation that adds the `lgtm` label.
+
+### Becoming a reviewer
+
+When you meet the
+[requirements](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer), you can become a SIG Docs reviewer. Reviewers in other SIGs must apply separately for reviewer status in SIG Docs.
+
+To apply:
+
+1. Open a pull request that adds your GitHub user name to a section of the
+[OWNERS_ALIASES](https://github.com/kubernetes/website/blob/master/OWNERS) file
```

Comment:
```suggestion
[OWNERS_ALIASES](https://github.com/kubernetes/website/blob/master/OWNERS_ALIASES) file
```

Reply:
I took a closer look and **OWNERS_ALIASES** will be what Prow uses to assign reviewers and https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml is what GitHub uses when you `@` a specific team (ex: `@sig-docs-en-reviews`).

Do you think it's worth including a note to PR the `teams.yaml` too?

Reply:
@jimangel It's only Leads who PR to `teams.yaml` right? Or should IC's be doing that too?